### DISCUSSION TITLE
Is it possible to use Supabase database only without using the auth?

### DISCUSSION THREAD

**Discussion by GeeWhizIamInTheMatrix** [supabase/supabase#19211] [Questions]
I already have an auth system in place using FirebaseAuth, but I wanted to use a relational database with my project. So is it possible to use Supabase postgress without Supabase Auth

**Comment by GaryAustin1** [comment]
Yes.  If you want to use RLS though you will need a jwt made with the same key as the Supabase jwt secret.
Then easier to do with a SUB claim in the jwt that is a UUID, but not required.
